assets/scss/04-templates/_details.scss

```diff
@@ -29,19 +29,8 @@
     .ma__sidebar-heading,
     .ma__comp-heading {
       @include ma-heading-colored;
-      border: none;
-    }
-  }
-}
-
-//theme
```

Comment:
Theme specific styles are separated from the common styles, so everybody knows where to touch for theme related changes.

Reply:
The same suggestion as the `assets/scss/03-organisms/_stacked-row-section.scss`'s.

- Remove LL. 34-35 `}`.
- Keep `//theme` at L. 37.
- Remove LL. 38-41
   ```
   .ma__details {
   
     .page-content > .ma__contact-list {
       
   ```
- Add `//theme` to L. 47